app/styles/library.scss

```diff
@@ -0,0 +1,7 @@
+button.signout {
```

Comment:
Is this necessary? Also, all the styling for the sign out button should be in `application.scss`.